guide/start/policies.md

```diff
@@ -355,8 +355,8 @@ Tomcat on the vagrant VMs named "byon1" to "byon4":
 
 {% highlight bash %}
 $ for i in byon{1..4}; do
-$   vagrant ssh ${i} --command 'ps aux | grep -i tomcat |  grep -v grep | awk '\''{print $2}'\'' | xargs kill -9'
```

Comment:
Where is our magic that used to treat "$" prefix in a special way (putting it into a separate div, so it wouldn't be included if someone highlighted the text to copy-paste it)? I can't find it.

@drigodwin said that he thinks it may have broken at around the same time as someone did a bootstrap upgrade.

@ahgittin @sjcorbett do you know where that code is?